pkg/stanza/fileconsumer/file.go

```diff
@@ -320,6 +320,11 @@ func (f *Input) loadLastPollFiles(ctx context.Context) error {
 		return fmt.Errorf("decoding file count: %w", err)
 	}
 
+	if knownFileCount > 0 {
+		f.Infow("Resuming from previously known offset(s). 'start_at' setting is not applicable.")
```

Comment:
Is there any way to override this feature?  Feels like there may be situations where a user would want to re-ingest a file, like if their collector didn't have the proper processors.  Probably not a production feature, but seems like a good option.

Reply:
Sort of. In the simplest case, the user just needs to remove or alter the storage extension. In the case of the `file_storage` extension, they can clear out the storage directory that they've configured, or just choose a new directory to stash offsets.